include/constants/pokemon.h

```diff
@@ -352,14 +352,18 @@
 #define EVO_SPECIFIC_MAP                  32     // Pokémon levels up on specified map
 #define EVO_LEVEL_NATURE_AMPED            33     // Pokémon reaches the specified level, it has a Hardy, Brave, Adamant, Naughty, Docile, Impish, Lax, Hasty, Jolly, Naive, Rash, Sassy, or Quirky nature.
 #define EVO_LEVEL_NATURE_LOW_KEY          34     // Pokémon reaches the specified level, it has a Lonely, Bold, Relaxed, Timid, Serious, Modest, Mild, Quiet, Bashful, Calm, Gentle, or Careful nature.
+#define EVO_SCRIPT_TRIGGER_DMG            36     // Pokémon has specified HP below max, then player interacts trigger
+#define EVO_DARK_SCROLL                   37     // interacts with Scroll of Darkness
+#define EVO_WATER_SCROLL                  38     // interacts with Scroll of Waters
```

Comment:
```suggestion
#define EVO_CRITICAL_HITS                 35     // Pokémon performs specified number of critical hits in one battle
#define EVO_SCRIPT_TRIGGER_DMG            36     // Pokémon has specified HP below max, then player interacts trigger
#define EVO_DARK_SCROLL                   37     // interacts with Scroll of Darkness
#define EVO_WATER_SCROLL                  38     // interacts with Scroll of Waters
```

Reply:
Just so you know, you avoid conflicts of this type by making resulting changes in the exact same way as the other branch you're doing. In this case, you need to add all 4 evolution method defines to both branches

Reply:
Got it! Wasn't sure exactly how to divide up the changes, it was all originally one branch.